geoh5py/groups/group.py

```diff
@@ -75,7 +75,11 @@ def add_comment(self, comment: str, author: str | None = None):
             self.comments.values = self.comments.values + [comment_dict]
 
     def copy_from_extent(
-        self, bounds: np.ndarray, parent=None, copy_children: bool = True
+        self,
+        bounds: np.ndarray,
+        parent=None,
+        copy_children: bool = True,
+        clear_cache: bool = False,
```

Comment:
**issue (doc):** obsolete doc of parameters

The 3 last parameters are not documented, while the doc mentions non existing parameter `attributes`